examples/b2b/app/components/Header.tsx

```diff
@@ -129,7 +137,6 @@ function HeaderMenuMobileToggle() {
 
 function SearchToggle() {
   const {open} = useAside();
-  const {publish, shop, cart, prevCart} = useAnalytics()
```

Comment:
this is just a mistake right? @wizardlyhel

Reply:
yea mistake - it should be on the `CartBadge` function right below